docs/index.rst

```diff
@@ -58,65 +26,56 @@ Install Lark
 
    $ pip install lark
 
-Syntax Highlighting
--------------------
 
--  `Sublime Text & TextMate`_
--  `Visual Studio Code`_ (Or install through the vscode plugin system)
--  `Intellij & PyCharm`_
--  `Vim`_
--  `Atom`_
+-------
+
+How the documentation is organized
+----------------------------------
+
+A high-level overview of how it’s organized will help you know where to look for certain things:
 
-.. _Sublime Text & TextMate: https://github.com/lark-parser/lark_syntax
-.. _Visual Studio Code: https://github.com/lark-parser/vscode-lark
-.. _Intellij & PyCharm: https://github.com/lark-parser/intellij-syntax-highlighting
-.. _Vim: https://github.com/lark-parser/vim-lark-syntax
-.. _Atom: https://github.com/Alhadis/language-grammars
+* Tutorials take you by the hand through a series of steps how to get familiar with Lark. Start here if you’re new to Lark.
+* How-to guides are bite-sized, problem-specific solutions to common tasks.
+* Addendums covers background information to Lark.
+* References contain syntax and semantics reference material for Lark.
```

Comment:
I personally don't really like if documentation feels the need to explain how it works. The naming of sections should be good enough that it's obvious. This is IMO the case for Tutorials, Addendum and References. How-to is a bit less clear, Recipes is a better name there IMO.

Even if we keep this list, the first sentence that explains that this is an explanations of sections is definitely not needed and just represents noise.

Reply:
That's a fair point. Actually, I had stolen this part from the Django documentation. Maybe for the Django docs it's justified as their work is huge.

I have no problem to remove it, but let's give it a bit more time to see what others think about this.